wicket-request/src/main/java/org/apache/wicket/request/mapper/info/PageComponentInfo.java

```diff
@@ -69,23 +69,18 @@ public ComponentInfo getComponentInfo()
 	@Override
 	public String toString()
 	{
-		StringBuilder result = new StringBuilder();
-		if (pageInfo != null)
-		{
-			result.append(pageInfo.toString());
-		}
+		String result = pageInfo.toString();
 		if (componentInfo != null)
 		{
-			result.append(SEPARATOR);
-			result.append(componentInfo);
+			result = result + SEPARATOR + componentInfo;
```

Comment:
The check `if (pageInfo != null)` is no more there.

Reply:
The constructor for the class ensures that this final field can not be null.